src/algebra/algebra/basic.lean

```diff
@@ -115,10 +118,14 @@ See the implementation notes in this file for discussion of the details of this
 @[nolint has_inhabited_instance]
 class algebra (R : Type u) (A : Type v) [comm_semiring R] [semiring A]
   extends has_smul R A, R →+* A :=
+[to_has_opposite_smul : has_smul Rᵐᵒᵖ A]
```

Comment:
I'd argue that it would be more convenient, adding a field `op_smul` and making `to_has_opposite_smul` an instance.

Reply:
The advantage of this spelling is that lean automatically picks up the instance as an instance argument if it already exists